enhancements/build-as-userns.md

```diff
@@ -0,0 +1,82 @@
+# Build containers images as user
+
+## Goals
+
+There is two different scenarios we want to achieve :
+
+1) Allow user to run buildah as root inside the container but running as user on the host
+
+2) Running as user inside the container.
```

Comment:
This is rootless (similar to what we can achieve on desktop/laptops with `podman`/`buildah`)